tests/python/gaia-ui-tests/gaiatest/apps/contacts/regions/contact_form.py

```diff
@@ -144,13 +144,16 @@ def __init__(self, marionette):
         update = Wait(self.marionette).until(expected.element_present(*self._update_locator))
         Wait(self.marionette).until(lambda m: update.location['y'] == 0 and update.is_displayed())
 
-    def tap_update(self, return_details=True):
+    def tap_update(self, return_class=None):
```

Comment:
I'd suggest to change `None` to `'ContactDetails'`. Returning ContactDetails was the default behavior, so tests like:
* `gaiatest/tests/functional/contacts/test_add_photo_to_contact.py`
* `gaiatest/tests/functional/contacts/test_edit_contact.py`
* `gaiatest/tests/functional/contacts/test_import_edit_export_contact.py` 
won't be broken.